bindings/nodejs/lib/wallet/Account.ts

```diff
@@ -716,21 +712,21 @@ export class Account {
 
     /**
      * Mint native tokens.
-     * @param nativeTokenOptions The options for minting tokens.
+     * @param mintNativeTokenParams The options for minting tokens.
      * @param transactionOptions The options to define a `RemainderValueStrategy`
      * or custom inputs.
      * @returns The minting transaction and the token ID.
      */
     async mintNativeToken(
-        nativeTokenOptions: NativeTokenOptions,
+        mintNativeTokenParams: MintNativeTokenParams,
         transactionOptions?: TransactionOptions,
     ): Promise<MintTokenTransaction> {
         const response = await this.methodHandler.callAccountMethod(
             this.meta.index,
             {
                 name: 'mintNativeToken',
                 data: {
-                    nativeTokenOptions: nativeTokenOptions,
+                    mintNativeTokenParams: mintNativeTokenParams,
```

Comment:
Shouldn't the field be only `params`? Also for the others below

Reply:
Yes, thank you. It's hard to find all the locations 😤